src/scripts/basicContext.js

```diff
@@ -51,14 +49,14 @@ const buildItem = function (item, num) {
 	// Generate item
 	if (item.type === ITEM) {
 		html = `
-		       <tr class='basicContext__item ${item.class}'>
-		           <td class='basicContext__data' data-num='${item.num}'>${span}${item.title}</td>
-		       </tr>
-		       `;
+				 <tr class='basicContext__item ${item.class}'>
+						 <td class='basicContext__data' data-num='${item.num}'>${span}${item.title}</td>
```

Comment:
Why do we suddenly have two tabs for each level of indention? IMHO, there should be _one_ tab per level? This applies here and for most of the code below.